WalletWasabi.Fluent/Models/FeeDisplayFormat.cs

```diff
@@ -6,6 +7,17 @@ public enum FeeDisplayFormat
 {
 	BTC,
 
-	[FriendlyName("sat/vByte")]
-	SatoshiPerByte,
+	[FriendlyName("sats")]
+	Satoshis,
 }
+
+public static class FeeDisplayFormatExtensions
```

Comment:
this might live here: https://github.com/zkSNACKs/WalletWasabi/tree/master/WalletWasabi.Fluent/Extensions